README.md

```diff
@@ -22,33 +21,53 @@ CMR-GraphQL is an API developed by [NASA](http://nasa.gov) [EOSDIS](https://eart
 
 ## Application Installation
 
-Before running the application you'll want to ensure that all necessary packages are installed by running:
+### Prerequisites
 
-   npm install
+- Docker
+- aws-sam-cli (`brew install aws-sam-cli`)
```

Comment:
I also needed into install dotenvx (brew install dotenvx/brew/dotenvx) and aws-cdk (brew install aws-cdk)

Reply:
Those should come from `npm install`, I don't have those installed with brew in my machine. You might need to wait for the @edsc/cdk-utils library to be published